lighthouse-core/audits/legacy-javascript.js

```diff
@@ -159,97 +160,102 @@ class LegacyJavascript extends Audit {
+      {module: 'es6.reflect.apply', name: 'Reflect.apply'},
+      {module: 'es6.reflect.construct', name: 'Reflect.construct'},
+      {module: 'es6.reflect.define-property', name: 'Reflect.defineProperty'},
+      {module: 'es6.reflect.delete-property', name: 'Reflect.deleteProperty'},
+      {module: 'es6.reflect.get', name: 'Reflect.get'},
+      {module: 'es6.reflect.get-own-property-descriptor', name: 'Reflect.getOwnPropertyDescriptor'},
+      {module: 'es6.reflect.get-prototype-of', name: 'Reflect.getPrototypeOf'},
+      {module: 'es6.reflect.has', name: 'Reflect.has'},
+      {module: 'es6.reflect.is-extensible', name: 'Reflect.isExtensible'},
+      {module: 'es6.reflect.own-keys', name: 'Reflect.ownKeys'},
+      {module: 'es6.reflect.prevent-extensions', name: 'Reflect.preventExtensions'},
+      {module: 'es6.reflect.set', name: 'Reflect.set'},
+      {module: 'es6.reflect.set-prototype-of', name: 'Reflect.setPrototypeOf'},
+      {module: 'es6.set', name: 'Set'},
+      {module: 'es6.string.code-point-at', name: 'String.prototype.codePointAt'},
+      {module: 'es6.string.ends-with', name: 'String.prototype.endsWith'},
+      {module: 'es6.string.from-code-point', name: 'String.fromCodePoint'},
+      {module: 'es6.string.includes', name: 'String.prototype.includes'},
+      {module: 'es6.string.raw', name: 'String.raw'},
+      {module: 'es6.string.repeat', name: 'String.prototype.repeat'},
+      {module: 'es6.string.starts-with', name: 'String.prototype.startsWith'},
+      {module: 'es6.string.trim', name: 'String.prototype.trim'},
+      {module: 'es6.typed.array-buffer', name: 'ArrayBuffer'},
+      {module: 'es6.typed.data-view', name: 'DataView'},
+      {module: 'es6.typed.float32-array', name: 'Float32Array'},
+      {module: 'es6.typed.float64-array', name: 'Float64Array'},
+      {module: 'es6.typed.int16-array', name: 'Int16Array'},
+      {module: 'es6.typed.int32-array', name: 'Int32Array'},
+      {module: 'es6.typed.int8-array', name: 'Int8Array'},
+      {module: 'es6.typed.uint16-array', name: 'Uint16Array'},
+      {module: 'es6.typed.uint32-array', name: 'Uint32Array'},
+      {module: 'es6.typed.uint8-array', name: 'Uint8Array'},
+      {module: 'es6.typed.uint8-clamped-array', name: 'Uint8ClampedArray'},
+      {module: 'es6.weak-map', name: 'WeakMap'},
+      {module: 'es6.weak-set', name: 'WeakSet'},
+      {module: 'es7.array.includes', name: 'Array.prototype.includes'},
+      {module: 'es7.object.entries', name: 'Object.entries'},
+      {module: 'es7.object.get-own-property-descriptors', name: 'Object.getOwnPropertyDescriptors'},
+      {module: 'es7.object.values', name: 'Object.values'},
+      {module: 'es7.string.pad-end', name: 'String.prototype.padEnd'},
+      {module: 'es7.string.pad-start', name: 'String.prototype.padStart'},
+      /* eslint-enable max-len */
+    ];
+  }
+
   /**
    * @return {Pattern[]}
    */
   static getPolyfillPatterns() {
-    return [
-      'Array.fill',
```

Comment:
what happened to `Array.fill`? there were 79 polyfills and now 78, got lucky that the very first one is the one that disappeared 😆

Reply:
Oh! it was because this should have originally been `Array.prototype.fill` but I made a mistake and wrote `Array.fill`. When I programmatically filtered this array that difference reared its head like this